README.md

```diff
@@ -119,11 +119,17 @@ brew tap robusta-dev/homebrew-krr
 brew install krr
 ```
 
```

Comment:
lowercase `f` in first please (it isn't not a new sentence so it shouldn't be capitalized)

And can we fix the existing typo in `successful` (not yours :)